tyk-pro/templates/deployment-dash.yaml

```diff
@@ -71,9 +71,9 @@ spec:
           - name: TYK_DB_TYKAPI_PORT
             value: "{{ .Values.gateway.service.port }}"
           {{- if .Values.gateway.control.enabled }}
-          - name: TYK_DB_TYKAPI_HOST
+          - name: TYK_GW_CONTROLAPIHOSTNAME
             value: "{{ include "tyk-pro.gwproto" . }}://gateway-control-svc-{{ include "tyk-pro.fullname" . }}.{{ .Release.Namespace }}"
-          - name: TYK_DB_TYKAPI_PORT
+          - name: TYK_GW_CONTROLAPIPORT
```

Comment:
environment variables are wrong based on the [docs](https://tyk.io/docs/tyk-oss-gateway/configuration/#control_api_hostname). Smoke tests were failing during upgrade due to this error.